### DISCUSSION TITLE
503 Error

### DISCUSSION THREAD

**Discussion by RobDobsonNC** [ManifoldScholar/manifold#3575] [Q&A] [answered]
Hello - first time with Manifold, and I've installed on a Lightsail instance. All seems well until the green bar at the top reaches the other side, and then it drops with a 503 error. 

I'm sure this has been seen before, but I can't see a fix. Could anyone help?

**Comment by zdavis** [comment]
I don't really have an answer for you on this, as we haven't tried installing Manifold on Lightsail before. If you can give us any kind of log output that describes what the problem is, we might be able to point you in the right direction, but Lightsail isn't a platform we test deployment on, so it's hard to give you an answer hewre.

**Reply by RobDobsonNC** [reply]
Fair enough! Which platform would you recommend, I'm fairly agnostic as far as this install goes

**Comment by zdavis** [comment]
See https://manifoldscholar.github.io/manifold-docusaurus/docs/administering/installation

We tend to recommend package installation on Ubuntu18 or 20.

**Comment by RobDobsonNC** [comment]
Yes I saw, I set up a Ubuntu install on a Lightsail instance. Works fine for about 30 seconds.

Which error logs can I share? I had a brief look. It couldn’t see anything unusual, though I’ve no experience with manifold.

Thanks!

Northern Comfort
Sent with Spark

**Comment by zdavis** [comment]
If you're installing the package from the command line, per the instructions, there is typically quite a bit of log output when you run the install command. We would need to see the output from running the `dpkg -i manifold_7.0.1-1_amd64.deb` command.

**Comment by RobDobsonNC** [comment, marked as answer]
Thanks @zdavis for your help, though I've got it working now. The issue appeared to be (my fault) I had the IP address within manifold.rb. Once I'd assigned a static IP and qualified name, and amended manifold.rb accordingly, worked fine. Thanks again, apologies for wasting your time.

**Comment by zdavis** [comment]
No apology necessary! Really glad to hear you got it working!
